lib/boom_notifier/error_info.ex

```diff
@@ -80,4 +79,35 @@ defmodule ErrorInfo do
     |> Tuple.to_list()
     |> Enum.join(".")
   end
+
+  @spec build_custom_data(map(), custom_data_strategy_type) :: map()
+  defp build_custom_data(_conn, :nothing), do: nil
```

Comment:
Should these be renamed to `build_metadata` to match the displayed name?

Reply:
Same as above